polyglot/piranha/src/lib.rs

```diff
@@ -11,9 +11,10 @@ Copyright (c) 2022 Uber Technologies, Inc.
  limitations under the License.
 */
 
+use config::CommandLineArguments;
 use models::{
-  piranha_arguments::PiranhaArguments, piranha_output::PiranhaOutputSummary,
-  source_code_unit::SourceCodeUnit,
+  piranha_arguments::PiranhaArguments,
+  piranha_output::PiranhaOutputSummary, source_code_unit::SourceCodeUnit,
```

Comment:
Shouldn't these be each in one line? (since they don't all fit in a single line)

Reply:
I ll run cargo format as a follow up